source/js/ellipsis.js

```diff
@@ -0,0 +1,56 @@
+/* global errorHandling:false */
+
+var EllipsisUtil = require("./text-ellipsis").EllipsisUtil;
```

Comment:
❓
The imports didn't work with 
```/* global EllipsisUtil:false errorHandling:false EllipseableObject:false */```

Reply:
it's not a good practice to polute the global namespace aka window (it's the same in any language).. It's a matter of time into a 3rd party comes and overwrite something you were expecting to be there (iAdvise/adobe omniture etc...) Dependencies should be scoped for its necessity (injected if possible)..

Reply:
Ok. Got it.